tests/lib/bin/session-tool

```diff
@@ -112,6 +121,18 @@ cat "$tmp_dir/ready.pipe" >/dev/null
 cat >"$tmp_dir/stdin.pipe" &
 cat <"$tmp_dir/stdout.pipe" >&1 &
 cat <"$tmp_dir/stderr.pipe" >&2 &
+(
+	echo "#!/bin/sh"
+	test -n "$pid_file" && echo "echo \$$ >\"$pid_file\""
+	printf "exec "
+	for arg in "$@"; do
+		printf '"%s" ' "$arg"
+	done
+	echo
+)>"$tmp_dir/exec"
+# cat "$tmp_dir/exec"
```

Comment:
Remove?

Reply:
Removed in the follow-up to this branch.